src/main/io/gps.c

```diff
@@ -969,6 +1470,31 @@ static void gpsNewData(uint16_t c)
         return;
     }
 
+    gpsPackageCounter += 1;
+    if (millis() - gpsPackageCounterTime >= 1000) {
```

Comment:
cmp32

Reply:
Variable is hard to interpret, maybe `gpsPackageCounterStartTime` and move it under `gpsData`
In any case, I really don't feel this is necessary compared to the `lastMessage` we already have - simplicity is crucial, and `gps.c` is already rather complex.

Reply:
I'll look into the various time stamps, there seem to be quite a few.